src/theme/dark.scss

```diff
@@ -0,0 +1,934 @@
+      background-color: $background-color-dark-theme;
+    }
+    .search-wrapper {
+      background-color: $background-secondary-dark-theme;
+      color: color($colors, light-grey);
+      .scan-icon {
+        border-left: 1px solid rgba(151, 151, 151, 0.2);
+      }
+    }
+    .invalid {
+      border: 1px solid color($colors, error);
+      color: color($colors, error);
+      background-color: #fff6f7;
+    }
+    .card {
+      background-color: $background-secondary-dark-theme;
+      ion-item-divider {
+        background-color: $background-secondary-dark-theme;
+        color: $text-color-dark-theme;
+      }
+    }
+  }
+
+  // Tab Cards ------
+  buy-card-page {
+    .amount-box {
+      background-image: linear-gradient(
+        -180deg,
+        $background-color-dark-theme 30%,
+        #101010 100%
+      );
+      .discount-header {
+        background-color: $background-secondary-dark-theme;
+        color: $text-color-dark-theme;
+      }
+    }
+    .buy-box {
+      background-color: $background-color-dark-theme;
+    }
+  }
+
+  // Tab Settings ------
+  page-settings,
+  page-about {
+    .bp-list {
+      ion-note.note {
+        background: $background-secondary-dark-theme;
+        color: color($colors, light-grey);
+      }
+      .icon-qrcode {
+        filter: invert(1);
+      }
+      .create-button {
+        color: color($colors, primary);
+      }
+    }
+  }
+  page-alt-currency,
+  page-key-settings,
+  page-wallet-information,
+  page-wallet-addresses,
+  page-wallet-settings {
+    .bp-list.settings-list {
+      ion-item.item,
+      button.item {
+        .item-inner {
+          ion-note.note {
+            background: $background-secondary-dark-theme;
+            color: color($colors, light-grey);
+          }
+        }
+      }
+    }
+  }
+  page-addressbook-add,
+  page-join-wallet {
+    ion-item {
+      background: $background-color-dark-theme;
+      color: color($colors, light-grey);
+    }
+  }
+  page-wallet-addresses {
+    button.item {
+      background: $background-color-dark-theme;
+    }
+  }
+
+  page-scan {
+    .page-scan-has-problems {
+      background-color: $toolbar-background-dark-theme;
+      .body {
+        background-color: $toolbar-background-dark-theme;
+        div {
+          color: $toolbar-background-dark-theme;
+        }
+      }
+    }
+  }
+
+  page-simplex {
+    .buy-crypto-button {
+      color: $text-color-dark-theme;
+    }
+  }
+  page-simplex-buy {
+    .bp-list {
+      ion-note.note {
+        .note-container {
+          color: $text-color-dark-theme;
+        }
+      }
+    }
+  }
+  page-add,
+  page-select-currency,
+  page-coin-selector,
+  select-invoice-page {
+    .list-button {
+      .item-title {
+        color: $text-color-dark-theme;
+      }
+      .item-subtitle {
+        color: color($colors, light-grey);
+      }
+    }
+  }
+  page-import-wallet,
+  page-create-wallet,
+  page-bitpay-phase-one-card-intro {
+    ion-item {
```

Comment:
We should cover all `ion-item` elements, not for specific page. Example of missing dark format (list of item from shapeshift): 

![Screen Shot 2020-03-10 at 16 47 24](https://user-images.githubusercontent.com/237435/76353207-11967400-62ef-11ea-89b6-de0bb23e9fb8.png)

Reply:
It would be enough to add the 'bp-list' class to <ion-list>. Good catch!